AppCenter/AppCenter/Internals/History/Device/MSDeviceTracker.m

```diff
@@ -21,6 +21,7 @@ @implementation MSDeviceTracker : NSObject
 
 static BOOL needRefresh = YES;
 static MSWrapperSdk *wrapperSdkInformation = nil;
+static NSString *overrideCountryCode = nil;
```

Comment:
Is it "override country code", not "overridden"? I see override as a verb and it looks more like a boolean variable than a code itself..

Reply:
`overriden` feels better to me yes.